src/store/getters/state-getters.ts

```diff
@@ -574,9 +552,9 @@ export const getDocumentOptionalEmail = (state: StateIF): string => {
   return state.stateModel.documentDelivery.documentOptionalEmail
 }
 
-/** Check for a 7 digit pattern to identify a Numbered company from the legal name. */
+/** Checks for a 7 digit pattern to identify a numbered company from the Legal Name. */
 export const isNumberedCompany = (state: StateIF): boolean => {
-  return RegExp('^\\d{7}$').test(getBusinessInformation(state)?.legalName?.split(' ')[0])
+  return RegExp('^\\d{7}$').test(getOriginalLegalName(state)?.split(' ')[0])
```

Comment:
The old code looked at the current legal name. The new code looks at the original legal name. Since this getter is only used in a very specific scenario, they were equivalent. However, the new code is more correct.